machineEnvironment.sh

```diff
@@ -34,7 +34,6 @@ scheduler=""     # none, slurm, pbs, ...
 queue=""         # standard queue to submit jobs to
 nthreads=""      # number of threads to use for parallel builds
 mpilaunch=""     # command to launch an MPI executable (e.g. aprun)
-installdir=""    # directory where libraries are installed
```

Comment:
Installdir is removed because managing the python environment is the responsibility of `daint_venv`, this had been a source of tight coupling between the three packages..